prime-router/src/main/kotlin/Report.kt

```diff
@@ -378,6 +380,10 @@ class Report {
         targetCounty: String? = null,
         metadata: Metadata,
     ): Report {
+        fun maybeSetString(row: Row, columnName: String, value: String) {
```

Comment:
Maybe a more descriptive name.  setRowValue?  setValueIfExists?  It is the maybe that confuses what the logic does.